### PR TITLE
add mobile docs to EUID (Android, iOS, GMA, IMA, CTV, Prebid)

### PR DESCRIPTION
**Pull request by genwhittTTD**
Add mobile docs. Update sidebar and all lead-in files.
NOTE: Mentions of UID2 are in code samples and links, for examination by Eng. Marking as draft pending that input.
List of docs added in this PR (excludes GMA Android):
1. Android SDK
2. IMA Android
3. IMA iOS
4. GMA iOS (but not Android until later)
5. Mobile integration Overview
6. Client-Side Integration Guide for Mobile
7. Client-Server Integration Guide for Mobile
8. CTV Integration Guide
9. Mobile Integration for Prebid Mobile SDK

### CODE REVIEW

File: docs/guides/integration-mobile-client-side.md [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+
+## Optional: EUID GMA/IMA Plugin for GAM Secure Signals integration
+
+<GMAIMA_Plugins />
+
+
+## Optional: EUID Prebid Mobile SDK Integration
+
+:::important
+The EUID Prebid Mobile SDK integration requires version 1.4.0 of the EUID SDK for Android, or version 1.5.0 of the EUID SDK for iOS.
```
Comment (caroline-ttd) [inline docs/guides/integration-mobile-client-side.md:582]: Let's update the version. The IOS sdk version is 1.7.0 and android sdk version is 1.5.0

File: docs/guides/integration-mobile-client-server.md [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+</Tabs>
+
+## Optional: EUID GMA/IMA Plugin for GAM Secure Signals integration
+
+<GMAIMA_Plugins />
+
+## Optional: EUID Prebid Mobile SDK Integration
+
+:::important
+The EUID Prebid Mobile SDK integration is for Android only, and requires version 1.4.0 of the SDK for Android.
```
Comment (caroline-ttd) [inline docs/guides/integration-mobile-client-server.md:306]: Let's update the version, android sdk version is 1.5.0

File: docs/guides/integration-mobile-client-side.md [hunk 3]
```diff
@@ -0,0 +1,585 @@
...
+
+1. [Optionally, integrate the EUID GMA/IMA Plugin for GAM Secure Signals integration](#optional-euid-gmaima-plugin-for-gam-secure-signals-integration).
+
+## Mobile SDK Version
+
+This guide provides instructions for using version 1.2.0 or higher of either of these EUID mobile SDKs:
+
+- SDK for Android
+- SDK for iOS
+
```
Comment (caroline-ttd) [inline docs/guides/integration-mobile-client-side.md:56]: Let's update the version. The IOS sdk version is 1.7.0 and android sdk version is 1.5.0

File: docs/guides/mobile-plugin-gma-ios.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+## Functionality
+
+This plugin simplifies integration with Google Mobile Ads (GMA) for any publishers who want to support EUID for apps running on iOS devices.
+
+## Version
+
+<!-- As of 2024-04-17 -->
+
+This documentation is for the EUID GMA Plugin for iOS version 0.3.2 or later.
```
Comment (caroline-ttd) [inline docs/guides/mobile-plugin-gma-ios.md:22]: IOS GMA version 1.0.0

File: docs/guides/mobile-plugin-gma-ios.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+]
+```
+
+### Installing with Xcode
+
+In the XCode user interface, under Package Dependencies, add the following entry for your apps:
+
+| Name | Location | Dependency Rule                         |
+| :--- | :--- |:----------------------------------------| 
+| uid2-ios-plugin-google-gma | `git@github.com:IABTechLab/uid2-ios-plugin-google-gma.git` | Exact Version: 0.3.2 |
```
Comment (caroline-ttd) [inline docs/guides/mobile-plugin-gma-ios.md:70]: IOS GMA version 1.0.0

File: docs/guides/mobile-plugin-ima-android.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+## Functionality
+
+This plugin simplifies integration with Google Interactive Media Ads (IMA) for any publishers who want to support EUID for apps running on Android devices.
+
+## Version
+
+<!-- As of 2023-07-15 -->
+
+This documentation is for the EUID IMA Plugin for Android version 0.5.0 and later.
```
Comment (caroline-ttd) [inline docs/guides/mobile-plugin-ima-android.md:22]: Android IMA version 1.5.0

File: docs/guides/mobile-plugin-ima-android.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+### Maven
+
+To install with Maven, add the SDK as a dependency in the `pom.xml` file:
+
+``` xml
+<dependency>
+  <groupId>com.uid2</groupId>
+  <artifactId>uid2-android-sdk-ima</artifactId>
+  <version>0.5.0</version>
```
Comment (caroline-ttd) [inline docs/guides/mobile-plugin-ima-android.md:68]: Android IMA version is 1.5.0

File: docs/snippets/_mobile_docs_prebid-mobile.mdx [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+   implementation("com.uid2:uid2-android-sdk-prebid:1.4.0")
+   ```
+   </TabItem>
+   <TabItem value='maven' label='Maven (Android)'>
+   To install with Maven, add the SDK as a dependency in the `pom.xml` file:
+   ```xml
+   <dependency>
+      <groupId>com.uid2</groupId>
+      <artifactId>uid2-android-sdk-prebid</artifactId>
+      <version>1.4.0</version>
```
Comment (caroline-ttd) [inline docs/snippets/_mobile_docs_prebid-mobile.mdx:35]: android version will be 1.5.0

File: docs/sdks/sdk-ref-android.md [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+
+### Installing with Maven 
+
+To install with Maven, add the SDK as a dependency in the `pom.xml` file:
+
+``` xml
+<dependency> 
+  <groupId>com.uid2</groupId> 
+  <artifactId>uid2-android-sdk</artifactId> 
+  <version>1.2.0</version>
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-android.md:104]: android version 1.5.0

File: docs/sdks/sdk-ref-android.md [hunk 10]
```diff
@@ -0,0 +1,214 @@
...
+## API Permissions
+
+To use this SDK, you'll need to complete the EUID account setup by following the steps described in the [Account Setup](../getting-started/gs-account-setup.md) page.
+You'll be granted permission to use specific functions offered by the SDK, and given credentials for that access.
+
+## SDK Version
+
+<!-- As of 7 May 2024 -->
+
+This documentation is for the SDK for Android version 1.2.0 and later.
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-android.md:51]: android version will be 1.5.0

File: docs/guides/mobile-plugin-ima-ios.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+]
+```
+
+### Installing with Xcode
+
+In the XCode user interface, under Package Dependencies, add the following entry for your apps:
+
+| Name | Location | Dependency Rule                         |
+| :--- | :--- |:----------------------------------------| 
+| uid2-ios-plugin-google-ima | `git@github.com:IABTechLab/uid2-ios-plugin-google-ima.git` | Exact Version: 0.3.2 |
```
Comment (caroline-ttd) [inline docs/guides/mobile-plugin-ima-ios.md:70]: IOS IMA version is 1.0.0

File: docs/guides/mobile-plugin-ima-ios.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+
+## Functionality
+
+This plugin simplifies integration with Google Interactive Media Ads (IMA) for any publishers who want to support EUID for apps running on iOS devices.
+
+## Version
+
+<!-- As of 2024-04-17 -->
+
+This documentation is for the EUID IMA Plugin for iOS version 0.3.2 or later.
```
Comment (caroline-ttd) [inline docs/guides/mobile-plugin-ima-ios.md:22]: IOS IMA version is 1.0.0

File: docs/guides/integration-mobile-client-server.md [hunk 13]
```diff
@@ -0,0 +1,309 @@
...
+4. [Configure the EUID mobile SDK](#configure-the-euid-mobile-sdk).
+5. [Check that the token was successfully generated and then pass it for bidstream use](#pass-generated-token-for-bidstream-use).
+6. [Optionally, integrate the EUID GMA/IMA Plugin for GAM Secure Signals integration](#optional-euid-gmaima-plugin-for-gam-secure-signals-integration).
+
+## Mobile SDK Version
+
+This guide provides instructions for using version 1.2.0 or higher of either of these EUID mobile SDKs:
+
+- SDK for Android
+- SDK for iOS
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/integration-mobile-client-server.md:53]: Done.

File: docs/guides/integration-mobile-client-side.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/guides/integration-mobile-client-side.md:582]: Done.

File: docs/guides/integration-mobile-client-server.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/guides/integration-mobile-client-server.md:306]: Done.

Comment (genwhittTTD) [reply docs/guides/integration-mobile-client-server.md:306]: Done.

File: docs/guides/integration-mobile-client-side.md (hunk 3 above)
Comment (genwhittTTD) [reply docs/guides/integration-mobile-client-side.md:56]: @caroline-ttd : Done but a question: is it version number or higher, per the original copy, or does it have to be the exact version number? Original docs are inconsistent... other places it lists the specific version numbers, only.

File: docs/guides/mobile-plugin-gma-ios.md (hunk 4 above)
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-gma-ios.md:22]: @caroline-ttd Done but should I also remove the "or later" here? I left it in for now.

File: docs/guides/mobile-plugin-ima-android.md (hunk 6 above)
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-ima-android.md:22]: @caroline-ttd Done. Also updated other instances. Please check.

File: docs/guides/mobile-plugin-ima-android.md (hunk 7 above)
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-ima-android.md:68]: Done.

File: docs/snippets/_mobile_docs_prebid-mobile.mdx (hunk 8 above)
Comment (genwhittTTD) [reply docs/snippets/_mobile_docs_prebid-mobile.mdx:35]: @caroline-ttd Done. Updated 2 instances in this file. Please check.

File: docs/sdks/sdk-ref-android.md (hunk 9 above)
Comment (genwhittTTD) [reply docs/sdks/sdk-ref-android.md:104]: Done.

File: docs/sdks/sdk-ref-android.md (hunk 10 above)
Comment (genwhittTTD) [reply docs/sdks/sdk-ref-android.md:51]: Done.

File: docs/guides/mobile-plugin-ima-ios.md (hunk 11 above)
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-ima-ios.md:70]: Done.

File: docs/guides/mobile-plugin-ima-ios.md (hunk 12 above)
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-ima-ios.md:22]: Done.

File: docs/guides/mobile-plugin-gma-ios.md [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+- [https://github.com/IABTechLab/uid2-ios-plugin-google-gma](https://github.com/IABTechLab/uid2-ios-plugin-google-gma)
+
+## Requirements 
+
+To run this plugin, install the following:
+
+1. Google Mobile Ads SDK v10.7.0 or later:
+   - [SDK](https://developers.google.com/admob/ios)
+   - [Release Notes](https://developers.google.com/admob/ios/rel-notes)
+1. SDK for iOS v0.3.2 or later:
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-gma-ios.md:37]: Done.

File: docs/guides/mobile-plugin-gma-ios.md [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+| Name | Location | Dependency Rule                         |
+| :--- | :--- |:----------------------------------------| 
+| uid2-ios-plugin-google-gma | `git@github.com:IABTechLab/uid2-ios-plugin-google-gma.git` | Exact Version: 0.3.2 |
+
+### Installing with CocoaPods
+
+Add the following entry in your `Podfile`:
+
+```
+pod 'UID2GMAPlugin', '0.3.2'
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-gma-ios.md:77]: Done.

File: docs/guides/mobile-plugin-ima-ios.md [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+-   [Xcode](#installing-with-xcode)
+-   [CocoaPods](#installing-with-cocoapods)
+
+### Installing with Package.swift
+
+Add the following dependency to Package.swift:
+
+```js
+dependencies: [
+  .package(url: "https://github.com/IABTechLab/uid2-ios-plugin-google-ima.git", exact: "0.3.2")
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-ima-ios.md:60]: Yep, got that one! Thx much.

File: docs/guides/mobile-plugin-ima-ios.md [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+| Name | Location | Dependency Rule                         |
+| :--- | :--- |:----------------------------------------| 
+| uid2-ios-plugin-google-ima | `git@github.com:IABTechLab/uid2-ios-plugin-google-ima.git` | Exact Version: 0.3.2 |
+
+### Installing with CocoaPods
+
+Add the following entry in your `Podfile`:
+
+```
+pod 'UID2IMAPlugin', '0.3.2'
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-ima-ios.md:77]: Yep, done, thx.

File: docs/guides/integration-mobile-client-side.md [hunk 18]
```diff
@@ -0,0 +1,585 @@
...
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+// Must be set before EUIDManager.shared is accessed
+UID2Settings.shared.environment = .custom(
+  url: URL(string: "https://global.prod.uidapi.com")!
+)
+// or use a named environment
+UID2Settings.shared.environment = .sydney
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/integration-mobile-client-side.md:263]: Done, thx! Fixed all instances of UID2Settings.shared.environment to UID2Settings.shared.euidEnvironment per our discussion.
Also fixed all locations Sydney and Singapore > London.

File: docs/snippets/_mobile_docs_prebid-mobile.mdx [hunk 19]
```diff
@@ -0,0 +1,114 @@
...
+
+2. Add the EUID Mobile SDK to your app, following the steps in [Add the EUID Mobile SDK to Your Mobile App](#add-the-euid-mobile-sdk-to-your-mobile-app).
+
+3. The EUID Prebid integration is distributed as a separate module, so you must add it as a dependency in your project. Follow the installation instructions that apply to your integration, out of the following options:
+
+   <Tabs groupId="dependency-selection">
+   <TabItem value='gradle' label='Gradle (Android)'>
+   Include the following in your Gradle configuration:
+   ```js
+   implementation("com.uid2:uid2-android-sdk-prebid:1.4.0")
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/snippets/_mobile_docs_prebid-mobile.mdx:26]: Yep, done, thx.

**Comment by caroline-ttd** [comment]
One small heads up, we probably need to update some links in the previous iOS sdk MR after this MR has been merged: https://github.com/European-Unified-ID/EUID-docs/pull/177

**Comment by sunnywu** [comment]
@caroline-ttd 
_One small heads up, we probably need to update some links in the previous iOS sdk MR after this MR has been merged: https://github.com/European-Unified-ID/EUID-docs/pull/177_

Why don't we just update the links in this MR altogether now?

**Comment by genwhittTTD** [comment]
@caroline-ttd + @sunnywu I believe I already covered those updates. All 3 of the files in https://github.com/European-Unified-ID/EUID-docs/pull/177 are in the current PR. I filled in some links I'd had to remove. LMK if you think I missed something. Thx.

File: docs/sdks/sdk-ref-javascript.md [hunk 20]
```diff
@@ -34,7 +34,7 @@ For integration steps for content publishers, see:
 For example applications with associated documentation, see:
   - The EUID Google Secure Signals with SDK v3 example:
     - [Code and docs](https://github.com/IABTechLab/uid2-web-integrations/tree/main/examples/google-secure-signals-integration/with_sdk_v3)
-    - Running site: [Client-Side EUID SDK Integration Example](https://secure-signals-jssdk-integ.uidapi.com/).
+    - Running site: [Client-Side UID2 SDK Integration Example](https://secure-signals-jssdk-integ.uidapi.com/).
```
Comment (caroline-ttd) [inline docs/sdks/sdk-ref-javascript.md:37]: @sunnywu Hi Sunny, do we have a Client-side EUID SDK Integration Example?

File: docs/guides/integration-mobile-client-server.md [hunk 21]
```diff
@@ -108,15 +108,15 @@ At this point, you are ready to use the EUID Identity generated server-side in t
...
+By default, the SDK is configured to work with the EUID production environment: `https://prod.euid.eu/v2`. If you want to use the EUID integration environment instead, provide the following URL in your call to EUIDManager initialization:
 
 <Tabs groupId="language-selection">
 <TabItem value='android' label='Android'>
 
 ```js
 EUIDManager.init(
   context = this,
-  serverUrl = "https://operator-integ.uidapi.com"
+  serverUrl = "https://integ.euid.eu/v2"
```
Comment (caroline-ttd) [inline docs/guides/integration-mobile-client-server.md:119]: @dcaunt Hi Dave, in the dev app, it has an [example](https://github.com/IABTechLab/uid2-android-sdk/blob/main/dev-app/src/main/java/com/uid2/dev/DevApplication.kt#L24-L27) that has been updated as below after merging the EUID support code.
```           
       EUIDManager.init(
                context = this,
                EUIDManager.Environment.Custom(EUID_INTEG_SERVER_URL),
                isLoggingEnabled = true,
        )
```
Do we need to update the code example in this doc as below?
```
       EUIDManager.init(
                context = this,
                EUIDManager.Environment.Custom("https://integ.euid.eu/v2")
        )
```

File: docs/guides/integration-mobile-client-server.md [hunk 22]
```diff
@@ -0,0 +1,309 @@
...
+
+By default, the SDK is configured to work with the EUID production environment: `https://prod.euid.eu/v2`. If you want to use the EUID integration environment instead, provide the following URL in your call to EUIDManager initialization:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+EUIDManager.init(
+  context = this,
+  serverUrl = "https://integ.euid.eu/v2"
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/integration-mobile-client-server.md:119]: Done + same mod in mobile-client-side line 222

File: docs/guides/integration-mobile-client-server.md [hunk 23]
```diff
@@ -0,0 +1,309 @@
...
+
+To specify an EUID server that is not the default, you can change it in the `init` call:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+EUIDManager.init(
+  context = this,
+  serverUrl = "https://prod.euid.eu/v2"
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/integration-mobile-client-server.md:156]: Done + same mod in mobile-client-side line 259.

**Review by caroline-ttd — APPROVED** [review]

File: docs/guides/integration-prebid-mobile-summary.md [hunk 24]
```diff
@@ -0,0 +1,19 @@
...
+description: Summary of information resources for setting up Prebid integration on mobile devices.
+hide_table_of_contents: false
+sidebar_position: 04
+---
+
+import Link from '@docusaurus/Link';
+
+# EUID Mobile Integration for Prebid Mobile SDK
+
+EUID integration with Prebid is also supported for Android mobile devices, using the Prebid Mobile SDK.
```
Comment (sunnywu) [inline docs/guides/integration-prebid-mobile-summary.md:14]: ios too now?

**Review by sunnywu — APPROVED** [review]

File: docs/guides/integration-mobile-client-server.md (hunk 23 above)
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/integration-mobile-client-server.md]: @dcaunt with apologies but I need a cross-check here. Your edit shows changing the code, but the URL shows as prod. I marked this as done in two files. But when I look at it now, in my files, the URL shows as integ. Also with a comma at EOL. I've now changed it to match the above. Not sure if I got it wrong, or something changed in the middle... but, anyway, please make sure it's now correct.

File: docs/guides/integration-prebid-mobile-summary.md (hunk 24 above)
Comment (genwhittTTD) [reply docs/guides/integration-prebid-mobile-summary.md]: Thx @sunnywu and @dcaunt . Updated in 2 places.

File: docs/guides/integration-mobile-client-server.md [hunk 25]
```diff
@@ -0,0 +1,309 @@
...
+4. [Configure the EUID mobile SDK](#configure-the-euid-mobile-sdk).
+5. [Check that the token was successfully generated and then pass it for bidstream use](#pass-generated-token-for-bidstream-use).
+6. [Optionally, integrate the EUID GMA/IMA Plugin for GAM Secure Signals integration](#optional-euid-gmaima-plugin-for-gam-secure-signals-integration).
+
+## Mobile SDK Version
+
+This guide provides instructions for using either of these EUID mobile SDKs:
+
+- SDK for Android (version 1.5.0 or later)
+- SDK for iOS (version 1.7 0 or later)
```
[the remark this answers is not in the document]
Comment (caroline-ttd) [reply docs/guides/integration-mobile-client-server.md]: Done, update to `SDK for iOS (version 1.7.0 or later)`

File: docs/guides/integration-mobile-client-server.md [hunk 26]
```diff
@@ -0,0 +1,309 @@
...
+  serverUrl = "https://operator-integ.uidapi.com"
+)
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+// Must be set before EUIDManager.shared is accessed
+UID2Settings.shared.environment = .custom(
```
[the remark this answers is not in the document]
Comment (caroline-ttd) [reply docs/guides/integration-mobile-client-server.md]: Done, updated to `euidEnvironment`

File: docs/sdks/sdk-ref-javascript.md (hunk 20 above)
Comment (caroline-ttd) [reply docs/sdks/sdk-ref-javascript.md]: I updated the link from UID2 SDK integration Example to EUID SDK integration example.

File: docs/guides/integration-mobile-client-server.md (hunk 23 above)
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/integration-mobile-client-server.md]: Many thanks @caroline-ttd for checking this for me!

**Pull request merged**